kfdef/kfdef_gcp_private_dex.yaml

```diff
@@ -0,0 +1,306 @@
+# Kfdef config to install Kubeflow with private GKE cluster.
```

Comment:
Should we follow the pattern in https://github.com/kubeflow/manifests/tree/master/kfdef to have a `kfdef_gcp_private_dex.v1.0.0.yaml`?

Same for the private_minimum.yaml

Reply:
How about publishing `v1.0.0` in separate PR, when the feature is more stable?

Reply:
> How about publishing v1.0.0 in separate PR

SG.

> when the feature is more stable

Do we expect more changes to this file or we just need to test it?

Reply:
Might need more change around configs